src/azure-cli/HISTORY.rst

```diff
@@ -30,6 +30,13 @@ Release History
 
 * Fix #11121: When using `az keyvault certificate list`, passing `--include-pending` now doesn't require a value of `true` or `false`
 
+**NetAppFiles**
+
+* Upgrade azure-mgmt-netapp to 0.7.0 to use API version 2019-08-01. This new API version includes:
+
+    -  Some volume properties associated with replication
```

Comment:
We don't need these detail information here. 😄

Reply:
Ok. I've updated to something more suitable I hope.

Reply:
@leonardbf Sorry for my unclear explanation. We need the information "Upgrade azure-mgmt-netapp to 0.7.0". Other part can be decided by yourself. 😄